app/controllers/api/v1/participant_declarations_controller.rb

```diff
@@ -75,6 +78,19 @@ def permitted_params
           raise
         end
       end
+
+      def log_schema_validation_results
+        errors = SchemaValidator.call(raw_event: request.raw_post)
+
+        if errors.blank?
+          Rails.logger.info "Passed schema validation"
+        else
+          Rails.logger.info "Failed schema validation for #{request.raw_post}"
+          Rails.logger.info errors
+        end
+      rescue StandardError => e
```

Comment:
I'm not hugely confident of this code so have added extra catch to prevent it from blocking calls